rpc_proxy.go

```diff
@@ -624,6 +632,20 @@ func (p *rpcProxy) convertSuperMacaroon(ctx context.Context, macHex string,
 	return nil, nil
 }
 
+// getPerms returns a map from URI to permissions required for the URI. The
+// first time it is called, the getPermsMap is used to set the contents of the
+// map. After that, the same map is returned.
+func (p *rpcProxy) getPerms() map[string][]bakery.Op {
```

Comment:
i hate that we have to use a lock here... but couldnt figure out a nicer way of doing it. Since the rpcProxy needs to be started _before_ the connection to LND is complete so that it is possible for the user to unlock lnd through the rpcProxy. But we have to connect to lnd before knowing its build tags. so couldnt pass a pre-set list to rpcProxy anymore...